pkg/snmp/traps/status.go

```diff
@@ -21,18 +23,37 @@ func init() {
 	trapsExpvars.Set("PacketsAuthErrors", &trapsPacketsAuthErrors)
 }
 
+func getDroppedPackets() int64 {
+	aggregatorMetrics, ok := expvar.Get("aggregator").(*expvar.Map)
+	if !ok {
```

Comment:
Unfortunately this is not an easy problem to solve, if there are 0 packets dropped we will face an error (because the expvar variables are not initialized yet).
But you're right that the reciprocal is wrong, seeing an error here does not _mean_ that there are zero packets dropped.

What I can do though is to hide this variable from the status if there are 0 dropped packets. Not a perfect solution, but will prevent seeing the wrong value.